microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase.yml

```
@@ -134,4 +134,4 @@ methods:
         type: any
         description: >-
           The value of the property after it is set.
-extends: <a href="https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.callablefunction.html">CallableFunction</a>
+extends: <a href="https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase">CallableFunction</a>

```

```suggestion
extends: <a href="https://learn.microsoft.com/microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase">CallableFunction</a>
```
omit locale per corp style guide

---

en-us/ removed.  Closing comment.  Dave is researching where CallableFunciton is defined.